Makefile

```diff
@@ -222,8 +214,40 @@ ifeq ($(quickstart), true)
 	./scripts/package_quickstart.sh deploy/$(target)/manifests/$(ver) deploy/$(target)/quickstart/olm.yaml deploy/$(target)/quickstart/crds.yaml deploy/$(target)/quickstart/install.sh
 endif
 
+################################
+#  OLM - Install/Uninstall/Run #
+################################
+
 .PHONY: run-console-local
 run-console-local:
 	@echo Running script to run the OLM console locally:
 	. ./scripts/run_console_local.sh
 
+.PHONY: uninstall
+uninstall:
+	@echo Uninstalling OLM:
+	- kubectl delete -f deploy/upstream/quickstart/crds.yaml
+	- kubectl delete -f deploy/upstream/quickstart/olm.yam
+	- kubectl delete catalogsources.operators.coreos.com
+	- kubectl delete clusterserviceversions.operators.coreos.com
+	- kubectl delete installplans.operators.coreos.com
+	- kubectl delete operatorgroups.operators.coreos.com subscriptions.operators.coreos.com
```

Comment:
I don't think this is needed since we're deleting the CRDs above, and that triggers GC for all of these CRs.